proposed/clock-meta.md

```diff
@@ -0,0 +1,154 @@
+# PHPDoc Meta Document
+
+## 1. Summary
+
+The purpose of using the ClockInterface is to provide a standard way to access the system 
+time, that would allow interopability when testing code that relies on the current time 
+rather than relying installing PHP extensions or use hacks like re-declaring the time() 
+function in other namespaces. 
+
+## 2. Why Bother?
+
+There are currently a few libraries that do provide the functionality on packagist, however 
+there is no interopability between these different libraries, as they ship with their own 
+clock interfaces. Symfony has a TimeMock library which uses namespace hacks to override the 
+`time()`, `date()`, `microtime()`, etc functions, however this does not solve mocking calls to 
+`new \DateTime()`
+
+Pros:
+
+* Consistent interface to get the current time
+* Easy to mock the wall clock time for repeatablility.
+
+Cons:
+
+* Extra overhead and developer effort to get the current time, not as simple as
+calling `time()` or `date()`.
+
+## 3. Scope
+
+### 3.1 Goals
+
+* Provide a simple and mockable way to read the current time
+* Allow interoperability between libraries when reading the clock
+
+### 3.2 Non-Goals
+
+* This PSR does not provide a recommendation on how and when to use the concepts
+  described in this document, so it is not a coding standard.
+* This PSR does not provide a reccomendation on how to handle timezones when 
+  retrieving the current time. This is left up to the implementation.
+
+## 4. Approaches
+
+### 4.1 Chosen Approach
+
+We have decided to formalize the existing practices, use by several other packages
+out in the wild. Some of the popular packages providing this functionality are: 
+lcobucci/clock, kreait/clock, ergebnis/clock, and mangoweb/clock. Some providing
+interfaces, and some relying on overloading (extending) the Clock class to mock the
+current time.
```

Comment:
What `Clock class` is it referring to?

Reply:
Like the wall clock - getting the current system time